config/secrets.defaults.yml

```diff
@@ -6,6 +6,7 @@ test:
     <<: *ibm_cloud_power_defaults
   ibm_cloud_vpc_defaults: &ibm_cloud_vpc_defaults
     api_key: IBM_CLOUD_VPC_API_KEY
+    provider_region: provider-region
```

Comment:
Does this have to be one of the regions that we declare in the `REGIONS` list?  I think this might fail on the regions validation

Reply:
Yes this would have to be one of the regions in https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/db/fixtures/ibm_cloud_vpc_regions.yml with the value of the `:name` key

Reply:
I haven't had this fail so far, where does the regions validation take place?

Reply:
Oh interesting it looks like there isn't a provider_region validation on VPC, that's a different issue though.

Most providers have `validates :provider_region, :inclusion => {:in => ->(_region) { ManageIQ::Providers::Amazon::Regions.names }}` which would fail the `create` if the region isn't in the list of "allowed" regions

Reply:
That's okay we can come back to this later